### ISSUE TITLE
Screen Reader: Sign in: The 'Licenses' link is not focused and announced

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77543]
## Action Performed:
1. Using Android TalkBack, open the Expensify app
2. Swipe and try to focus the 'Licenses' link in the footer

## Expected Result:
The link should be focused and announced with a link role and its visible name 'Licenses'.

## Actual Result:
When a user navigates using TalkBack swipes, the "Licenses" link is skipped in the focus sequence. As a result, the user receives no auditory indication that the link exists, rendering the content inaccessible.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure that accessible={true} is set on the link to make it focusable.
https://reactnative.dev/docs/accessibility

Use the link role.
https://reactnative.dev/docs/accessibility#accessibilityrole

## Environment:
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)

## Screenshots/Videos

![Image](https://github.com/user-attachments/assets/277f2a50-5546-4f43-acad-e4b1193717fc)

https://github.com/user-attachments/assets/32f526f2-11ca-4485-96fc-fab1297a107a

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005374
**Issue reported by:** NA
**Slack conversation:** NA
